xacro/__init__.py

```diff
@@ -200,12 +200,12 @@ def expose(*args, **kwargs):
 
     deprecate_msg = 'Using {name}() directly is deprecated. Use {ns}.{name}() instead.'
     # This is the list of symbols we have exposed for years now. Continue exposing them directly
-    expose('list', 'dict', 'map', 'len', 'str', 'float', 'int', 'True', 'False', 'min', 'max', 'round',
+    expose('list', 'dict', 'map', 'len', 'str', 'float', 'int', 'bool', 'True', 'False', 'min', 'max', 'round',
```

Comment:
Please omit from this list, which exposes the symbols into the _global_ namespace.
```suggestion
    expose('list', 'dict', 'map', 'len', 'str', 'float', 'int', 'True', 'False', 'min', 'max', 'round',
```

Reply:
The way it's currently phrased on the [wiki](https://github.com/ros/xacro/wiki#available-functions-and-symbols), it explicitly states that it is in the global namespace .

I'm fine with removing it, but I thought it made sense since it is a conversion function like `int`, `float` and `str`.

Reply:
Formally, the symbol was not exposed globally in the past and therefore should not be included in this list. Anyway, I agree with your argumentation: it is more convenient to use `bool()` directly, instead of `python.bool()`.
Given that `bool` was probably not used as a property variable name frequently, I'm also fine adding it to the global namespace.